src/hearings/containers/request-hearing/hearing-edit-summary/hearing-edit-summary.component.ts

```diff
@@ -251,6 +269,32 @@ export class HearingEditSummaryComponent extends RequestHearingPageFlow implemen
     return this.areObjectsfunctionallyDifferentCheck(hearingRequestMainModel, hearingRequestToCompareMainModel);
   }
 
+  private standardiseHearingDates(hearingWindow: HearingWindowModel) {
+    const standardisedHearingWindow: any = {};
+
+    if (hearingWindow.dateRangeStart) {
+      standardisedHearingWindow.dateRangeStart = moment(hearingWindow.dateRangeStart).format(HearingDateEnum.DefaultFormat);
+    }
+
+    if (hearingWindow.dateRangeEnd) {
+      standardisedHearingWindow.dateRangeEnd = moment(hearingWindow.dateRangeEnd).format(HearingDateEnum.DefaultFormat);
+    }
+
+    if (hearingWindow.firstDateTimeMustBe) {
+      standardisedHearingWindow.firstDateTimeMustBe = moment(hearingWindow.firstDateTimeMustBe).format(HearingDateEnum.DefaultFormat);
+    }
+
+    for (const key of Object.keys(hearingWindow)) {
```

Comment:
i think this works, but you could do the same by doing
standardisedHearingWindow = { ... hearingWindow }
//lines 275 - 285 here